packages/taucmdr/tests/__init__.py

```diff
@@ -201,6 +201,7 @@ def reset_project_storage(self, init_args=None):
             prefix = PROJECT_STORAGE.prefix
         except StorageError:
             pass
+
```

Comment:
For future reference, unless this change improves the pylint score, my preference is to try to avoid style & whitespace only changes in commits and pull requests dealing with larger, more substantive changes.